web/src/user_group_edit_members.ts

```diff
@@ -373,8 +509,57 @@ function remove_member({
     do_remove_user_from_group();
 }
 
+function remove_subgroup({
+    group_id,
+    target_subgroup_id,
+    $list_entry,
+}: {
+    group_id: number;
+    target_subgroup_id: number;
+    $list_entry: JQuery;
+}): void {
+    const group = user_groups.get_user_group_from_id(current_group_id);
+    if (!group) {
+        return;
+    }
```

Comment:
What does this case represent on the UI side?

Reply:
I thought this is present for avoiding the typescript errors in case when `get_user_group_from_id` returns undefined for invalid IDs but that throws error for such cases, so can remove it.